src/subdomains/core/buy-crypto/routes/buy/dto/pdf.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class PdfDto {
+  @ApiProperty({ description: 'Base64 encoded PDF' })
+  pdfBase64: string;
```

Comment:
Is `Base64` in the name really needed? 😅 Or could we do e.g. `pdfData`? The Info of the Base64 Endcoding is already in the description